src/redux/actions/organisationUnits.js

```diff
@@ -0,0 +1,30 @@
+import { getOrganisationUnits } from '../../utils/api'
+
+export const ACTION_TYPES = {
+    ORGANISATION_UNITS_RECEIVED: 'ORGANISATION_UNITS_RECEIVED',
+    ORGANISATION_UNITS_ERRORED: 'ORGANISATION_UNITS_ERRORED',
+}
+
+export const loadOrganisationUnitsSuccess = periodTypes => ({
+    type: ACTION_TYPES.ORGANISATION_UNITS_RECEIVED,
+    payload: periodTypes,
+})
+
+export const loadOrganisationUnitsError = error => ({
+    type: ACTION_TYPES.ORGANISATION_UNITS_ERRORED,
+    payload: error,
+})
+
+const loadOrganisationUnits = () => async dispatch => {
```

Comment:
I thought we agreed on using then/catch chaining. Do you prefer using async with try/catch?